### PR TITLE
Update for CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gr2m**
about submitting and reviewing pull-request

### CODE REVIEW

**Comment by gr2m** [comment]
ping @Charlotteis @hoodiehq/maintainers

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,10 +84,10 @@ implementing features, refactoring code), otherwise you risk spending a lot of
 time working on something that the project's developers might not want to merge
 into the project.
 
-Please adhere to [JavaScript Standard Style](https://github.com/feross/standard) and any other requirements (such as test coverage).
+### For new Contributors
 
-Adhering to the following this process is the best way to get your work
-included in the project:
+If you never created a pull request beofre, welcome :tada: :smile: [Here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```
Comment (varjmes) [inline CONTRIBUTING.md]: "beofre" should be "before"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -177,16 +203,43 @@ If you have commit access, please follow this process for merging patches and cu
...
+3. Check that a change has any necessary documentation.
+4. If there is anything you don’t like, leave a comment below the respective
+   lines and submit a "Request changes" review. Repeat until everything has
+   been addressed.
+5. If you are not sure about something, mention `@hoodie/maintainers` or specific
+   people for help in a comment.
+6. If there is only a tiny change left before you can merge it and you think
+   it’s best to fix it yourself, you can directly commit to the author’s fork.
+   Leave a comment about it so the author and others will know.
+7. Once everything looks good, add a "Approve" review. Don’t forget to say
```
Comment (varjmes) [inline CONTRIBUTING.md]: "add a "Approve" review" should be "add an "Approve" review"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -84,10 +84,10 @@ implementing features, refactoring code), otherwise you risk spending a lot of
 time working on something that the project's developers might not want to merge
 into the project.
 
-Please adhere to [JavaScript Standard Style](https://github.com/feross/standard) and any other requirements (such as test coverage).
```
Comment (varjmes) [inline CONTRIBUTING.md:87]: How comes we removed this line?

**Review by varjmes — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (gr2m) [reply CONTRIBUTING.md:87]: I was thinking it’s not needed to point it out, as all our tests include linting with `standard` anyway ... do you think we should leave it in anyway?

Comment (varjmes) [reply CONTRIBUTING.md:87]: Nope, leaving it out sounds good to me!

**Review by varjmes — APPROVED** [review]

**Pull request merged**